src/loader/LoadDataAction.cpp

```diff
@@ -113,93 +113,93 @@ LoadDataAction::LoadDataAction(std::string folder, float* res) : m_folder(folder
 //	cv::imwrite(last_dir->filename().string() + "_slices" + std::to_string(z_slices) + resolution + ".png", image_out, compression_params);
 //}
```

Comment:
Delete? Lots of commented out code above it as well

Reply:
I removed the old code.